db/migrate/20221107195522_move_existing_iso_datastore_records.rb

```diff
@@ -0,0 +1,27 @@
+class MoveExistingIsoDatastoreRecords < ActiveRecord::Migration[6.1]
+  class Storage < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+
+    self.inheritance_column = :_type_disabled
+  end
+
+  class IsoDatastore < ActiveRecord::Base
+  end
+
+  def up
+    datastore_class_name = "ManageIQ::Providers::Ovirt::InfraManager::IsoDatastore"
```

Comment:
We'll need to handle `Ovirt::InfraManager` and `Redhat::InfraManager`.  I imagine the majority will be `Redhat::InfraManager` but we did release with the `Ovirt::InfraManager` so it is possible a customer has some ISO Datastores on this.

You can join on a where like (totally untested):
```
IsoDatastore.in_my_region.
            .joins(:ext_management_system)
            .where(:ext_management_systems => {:type => "ManageIQ::Providers::Ovirt::InfraManager"})
```

Reply:
- Do we think there will be a lot of these? I'm guessing no, but if so, I can help come up with the sql for this.

@agrare did you say we'll need to create 2 possible `datastore_class_name` values based upon the ems?

```ruby
IsoDatastore.in_my_region
            .joins(:ext_management_system)
            .where(:ext_management_systems => {:type => ["ManageIQ::Providers::Ovirt::InfraManager", "ManageIQ::Providers::Redhat::InfraManager"})
            .each do |datastore|
  datastore_class_name = "#{datastore.ems.type}::IsoDatastore"
  Storage.create!(:ems_id => datastore.ems_id, :type => datastore_class_name)
end
```

Reply:
> Do we think there will be a lot of these? I'm guessing no, but if so, I can help come up with the sql for this.

Only Ovirt/RHV support IsoDatastores, so there are only two possible values